### ISSUE TITLE
[Masthead]: Web Components - VoiceOver cannot read Profile button and menu

### ISSUE THREAD

**Issue by IgnacioBecerra** [carbon-design-system/carbon-for-ibm-dotcom#8087]
### Description

When using VoiceOver, it seems that the VO focus gets stuck on the Search button and fails to read any of the profile button/menu. Arrow navigation fails to read it either. Tabbing once more makes the page lose focus and read the browser elements instead.

https://user-images.githubusercontent.com/24970122/150237909-d7965a88-02ef-4103-af76-40d55c8d04bd.mov

### Component(s) impacted

Masthead

### Browser

Chrome

### Carbon for IBM.com version

1.30.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

N/A

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

N/A

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by jeffchew** [comment]
Closing, can no longer replicate.
